zone/zoning.cpp

```diff
@@ -467,6 +467,8 @@ void Client::DoZoneSuccess(ZoneChange_Struct *zc, uint16 zone_id, uint32 instanc
 	//Force a save so its waiting for them when they zone
 	Save(2);
 
+	lock_save_position = true;
```

Comment:
This would need to be reset if world responds with an error (zone failed to bootup for example, where zoning gets cancelled)

Reply:
Updated